editor/src/components/canvas/canvas-strategies/strategies/grid-move-absolute.ts

```diff
@@ -215,6 +215,19 @@ function runGridMoveAbsolute(
     ]
   }
 
+  function getContainingRect(): CanvasRectangle {
+    if (selectedElementMetadata.specialSizeMeasurements.immediateParentProvidesLayout) {
+      const gridCellGlobalFrame = getGlobalFrameOfGridCell(gridCellGlobalFrames, targetRootCell)
+      return zeroRectIfNullOrInfinity(gridCellGlobalFrame)
+    } else {
+      const closestOffsetGlobalFrame = MetadataUtils.findElementByElementPath(
+        jsxMetadata,
+        selectedElementMetadata.specialSizeMeasurements.closestOffsetParentPath,
+      )?.globalFrame
```

Comment:
`selectedElementMetadata.specialSizeMeasurements.coordinateSystemBounds` should give you this info!

Reply:
we should rename it to offsetParentBounds for readability...

Reply:
furthermore, it also works if the selected element's offsetParent does not have metadata because it is internal to an unfocused component

Reply:
Fixed in 8b8a5e8a7.

Reply:
I believe you did not push 8b8a5e8a7 !

Reply:
My mistake, fixed now.